packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,36 +1,47 @@
 {
-  "finalSystemOwner": "DUMMY",
-  "controller": "DUMMY",
+  "numDeployConfirmations": 1,
+
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "controller": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+
+  "l1StartingBlockTag": "FILL_IN_DAY_OF",
 
-  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
   "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1200,
+  "maxSequencerDrift": 600,
```

Comment:
fine for this upgrade, but in the future can we update these values to include units? `maxSequencerDriftSeconds` is way more clear